contracts/Lock.sol

```diff
@@ -27,9 +27,9 @@ contract Lock is AragonApp, IForwarder, IForwarderFee {
     ERC20 public token;
     uint256 public lockDuration;
     uint256 public lockAmount;
-    uint256 public griefingFactor;
 
-    uint256 private constant WHOLE_GRIEFING = 100;
+    uint256 public spamPenaltyFactor;
+    uint256 private constant WHOLE_SPAM_PENALTY = 100;
```

Comment:
We could rename this to `PERCENTAGE_BASE`, as that's what's used in the Voting app. Up to you though, it's a hard term to define.

Reply:
That will be done in a separate PR  i'm doing today.